### DISCUSSION TITLE
Load testing with schemathesis

### DISCUSSION THREAD

**Discussion by Kludex** [schemathesis/schemathesis#2476] [Q&A]
Hi there 👋 

Is it possible to leverage `schemathesis` for load testing? I want the scenarios to be created from the OpenAPI, and then perform a load test on my app.

**Comment by Stranger6667** [comment]
Hey!

Yes, it is possible! A few notes on the workflow, which Schemathesis is designed for:

- It generates a test case then sends it to the app and uses the response as feedback to the generation engine. It is possible to generate them separately though
- Using `--target=response_time` would guide generation so new test cases are more likely to cause slower responses
- There is no time limit for the test run, only the maximum cap of examples per endpoint

In other words, there are some limitations on the config, but I know that some folks used it in such a way :) Please, let me know if such workflow would fit your expectations! If not, I'd be happy to discuss ways to make Schemathesis more fitting in into this :)
